~~~PATCH 10963a703a4b5a6c3ab2bb4a32d6fd0d:81400a9ed8792d8ddf96835da2a98ace lib/Foswiki/Configure/Checker.pm (Foswiki-1.1.3)
diff --git lib/Foswiki/Configure/Checker.pm lib/Foswiki/Configure/Checker.pm
--- lib/Foswiki/Configure/Checker.pm
+++ lib/Foswiki/Configure/Checker.pm
@@ -457,7 +457,11 @@ sub checkRCSProgram {
             $err .= $this->ERROR( $prog
                   . ' did not return a version number (or might not exist..)' );
         }
-        if ( $version =~ /^\d/ && $version < $rcsverRequired ) {
+
+        # Item11955 - '5.8.1' < 5.7 results in a warning (non-numeric compare)
+        # Best practice is to use CPAN:version, but isn't core until perl 5.10.
+        # So instead let's make the comparison work by stripping out sub-decimal
+        if ( $version =~ /\D(\d+(\.\d+)?)/ && $1 < $rcsverRequired ) {
 
             # RCS too old
             $err .=
